8.1/Dockerfile.c9s

```diff
@@ -0,0 +1,85 @@
+FROM quay.io/sclorg/s2i-base-c9s:c9s
+
+# This image provides an Apache+PHP environment for running PHP
+# applications.
+
+EXPOSE 8080
+EXPOSE 8443
+
+# Description
+# This image provides an Apache 2.4 + PHP 7.4 environment for running PHP applications.
+# Exposed ports:
+# * 8080 - alternative port for http
+
+ENV PHP_VERSION=8.0 \
```

Comment:
Wrong version. It should be `8.1`

Reply:
As 8.1 is already released we will not be building c9s version at all. The Dockerfile is there for the sole reason of not losing history for it.

Reply:
It is misleading to have `8.1/Dockerfile.c9s` and inside wrong PHP version.

Reply:
I do not see how it could be misleading to have old versions in unmaintained files (its not like we test them), but since it is a simple change to make I added both in a recent commit.